<launch>
  <!-- For logging -->
  <!-- <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time}] [${node}]: ${message}"/> -->

  <!-- For terminal (human) reading -->
  <env name="ROSCONSOLE_FORMAT" value="${message}"/>
  <!-- Essential parameters -->
  <param name="rt1a3_action_timeout" type="double" value="60.0"/>
  <param name="rt1a3_brake_threshold" type="double" value="1.0"/>

  <!-- Essential Nodes -->
  <node name="rt1a3_ui" pkg="rt1-assignment3"
    type="rt1a3_ui" required="false" output="screen"/>
  <node name="rt1a3_inputasync" pkg="rt1-assignment3"
    type="rt1a3_inputasync" required="false" output="screen"/>

  <node name="rt1a3_controller" pkg="rt1-assignment3" output="screen"
    type="rt1a3_controller" launch-prefix="gnome-terminal -- sh -c"/>
</launch>
